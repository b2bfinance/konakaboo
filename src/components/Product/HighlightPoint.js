import React from 'react';
import styled from 'styled-components';
import { queries } from '../../utils/media';

export const Wrapper = styled.li`
  font-size: 1.5rem;
  font-weight: ${props => props.theme.mainBoldFontWeight};

  ${queries.desktop`
    padding-right: 1rem;
  `};
`;

export default ({ points, theme }) =>
  points.map((point, i) => (
    <Wrapper key={i} theme={theme}>
      {point}
    </Wrapper>
  ));
