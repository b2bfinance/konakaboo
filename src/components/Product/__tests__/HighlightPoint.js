import React from 'react';
import { render } from 'enzyme';
import HighlightPoint from '../HighlightPoint';

describe('HighlightPoint', () => {
  test('renders correctly', () => {
    expect(
      render(<HighlightPoint points={['foo', 'bar']} theme={stubData.theme} />)
    ).toMatchSnapshot();
  });
});
