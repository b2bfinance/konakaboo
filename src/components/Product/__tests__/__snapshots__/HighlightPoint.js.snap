// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighlightPoint renders correctly 1`] = `
Array [
  .c0 {
  font-size: 1.5rem;
  font-weight: 400;
  margin-bottom: 0.5rem;
}

@media (min-width:62em) {
  .c0 {
    padding-right: 1rem;
  }
}

<li
    class="c0"
>
    foo
</li>,
  .c0 {
  font-size: 1.5rem;
  font-weight: 400;
  margin-bottom: 0.5rem;
}

@media (min-width:62em) {
  .c0 {
    padding-right: 1rem;
  }
}

<li
    class="c0"
>
    bar
</li>,
]
`;
